Applications/LLMEval/ViewModels/DeviceStat.swift

```diff
@@ -3,33 +3,22 @@ import LLM
 import MLX
 
 @Observable
-class DeviceStat {
+final class DeviceStat: @unchecked Sendable {
+
+    @MainActor
     var gpuUsage = GPU.snapshot()
-    private var initialGPUSnapshot = GPU.snapshot()
+
+    private let initialGPUSnapshot = GPU.snapshot()
     private var timer: Timer?
```

Comment:
Although this is a var it is only that way for the purpose of `init()` and the need to capture self.